### ISSUE TITLE
loxberry v3.0.0.2 integrated mqtt not detected

### ISSUE THREAD

**Issue by jonasv05** [Liver64/LoxBerry-Owntracks#2]
Hello, the plugin is no longer working since version 3.0 with integrated mqtt.
Is it possible to check for the integrated mqtt besides the plugin?
Thank you!

**Comment by Liver64** [comment]
You are right. I will have a look and provide an update. May be it takes while because I'm busy.

**Comment by Liver64** [comment]
fixed long time ago
